core/src/test/scala/integration/kafka/api/CustomQuotaCallbackTest.scala

```diff
@@ -179,7 +183,26 @@ class CustomQuotaCallbackTest extends IntegrationTestHarness with SaslSetup {
 
   private def createTopic(topic: String, numPartitions: Int, leader: Int): Unit = {
     val assignment = (0 until numPartitions).map { i => i -> Seq(leader) }.toMap
-    TestUtils.createTopic(null, topic, assignment, servers)
+    val adminZkClient = new AdminZkClient(null)
```

Comment:
this test is disabled, so we can just comment the code `TestUtils.createTopic(null, topic, assignment, servers)`